website/docs/reference/dbt-jinja-functions/builtins.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "builtins"
 id: "builtins"
-description: "The `builtins` variable references the builtin dbt context methods."
+description: "Read this guide to understand the builtins Jinga function in dbt."
```

Comment:
`Jinga` should be `Jinja`

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/builtins

content="Read this guide to understand the builtins Jinja function in dbt."